### PR TITLE
Add test "Uppercase letter in the middle of the word"

### PR DESCRIPTION
**Pull request by gestap**

### CODE REVIEW

File: src/alliteratedActors.test.js [hunk 1]
```diff
@@ -35,6 +35,14 @@ test('Ten people', () => {
     .toEqual('Vnnchyxwx Vxot');
 });
 
+test('Uppercase letter in the middle of the word', () => {
+  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Wjuybt Hc Fiwyr',
```
Comment (roman-mate) [inline src/alliteratedActors.test.js]: 1. can't find uppercase letter in the middle of the word in this test case. 
2. let's rephrase each description to explain what scenario is being tested. Maybe half of them are not really needed and they are testing essentially same scenarios.

Comment (gestap) [reply src/alliteratedActors.test.js]: done

File: src/alliteratedActors.test.js [hunk 2]
```diff
@@ -2,61 +2,41 @@
...
     ['Fpzwpgpq Vswaed Eagbsm', 'Qv Owj', 'Wzucu Qmwrw', 'Djnrai Uvrwyh',
       'Naclcthdv Bpooabnvt Gwpfsy']))
     .toEqual(undefined);
 });
 
-test('Ten people', () => {
-  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Wjuybt Hc Fiwyr',
-    'Vpikwvhgj Kt', 'Epuq Jji', 'Ohp Vt', 'Ufqmup Gdqhwwrv',
+test('Should work with three-word name', () => {
+  expect(alliteratedActors(['Wn Tjuyol', 'Eyd Gjvxav', 'Hjuybt Hc Fiwyr',
```
Comment (roman-mate) [inline src/alliteratedActors.test.js]: you can shorten this test case significantly to 
```
expect(alliteratedActors(['Ohp Op Ot', 'Vnnchyxwx Vxot'])
     .toEqual('Ohp Op Ot');
```

File: src/alliteratedActors.test.js (hunk 1 above)
Comment (roman-mate) [reply src/alliteratedActors.test.js]: shorten input array to minimal possible input to test scenario 'Should work only with first letter uppercase'.

File: src/alliteratedActors.test.js [hunk 3]
```diff
@@ -2,61 +2,41 @@
...
+      'Vpikwvhgj Kt', 'Epuq Jji', 'Ohp Vt', 'Ufqmup udqhUwrv',
+      'Ydbwtqwfl Rmjfebpn', 'Qghj Ujc', 'Amy Adams'
+    ]))
+    .toEqual('Amy Adams');
 });
 
-test('Four-word name', () => {
+test('Should work with four-word name', () => {
   expect(alliteratedActors(
     ['Rgrliwb Rb Ratting Qua', 'Plzgmcsw Hhda',
```
Comment (roman-mate) [inline src/alliteratedActors.test.js]: Shorten input to minimal possible to test the scenario.

File: src/alliteratedActors.test.js (hunk 1 above)
Comment (gestap) [reply src/alliteratedActors.test.js]: done

File: src/alliteratedActors.test.js (hunk 2 above)
Comment (gestap) [reply src/alliteratedActors.test.js]: done

File: src/alliteratedActors.test.js (hunk 3 above)
Comment (gestap) [reply src/alliteratedActors.test.js]: done

**Pull request merged**